webplugin/js/app/kommunicate-ui.js

```diff
@@ -404,14 +404,46 @@ setAvailabilityStatus : function (status){
 showClosedConversationBanner  : function(isConversationClosed){
     var messageText = MCK_LABELS["closed.conversation.message"];
     var conversationStatusDiv = document.getElementById("mck-conversation-status-box");
-    if (isConversationClosed){
+    var isCSATenabled = kommunicate._globals.collectFeedback;
+    var FEEDBACK_UPDATE_URL = "/feedback"
```

Comment:
@akshatdhabi if it is getting used only once then this can be removed.

Reply:
@devashishmamgain  done